### DISCUSSION TITLE
Ability to specify a project subdirectory to hold all the Playtime clips

### DISCUSSION THREAD

**Discussion by Chianti** [helgoboss/helgobox#1073] [Ideas]
### Rationale
Some users may organise their project into subdirectories, most of which is done automatically.
For instance, in my case:
· Manual backups in a folder called '- Backups'
· Auto Backups in a folder named '- AutoBackup(24hrs)'
· Project Renders in a folder called '- Project Renders'.
· Reapeaks in a folder called '- Reapeaks'.

Due to the '- 'sign, all these folders appear consistently and neatly at the top of every project folder in Windows Explorer.

Then, instead of dumping all recorded media into the root of the project folder, using 'Prefs > Recording > Recorded filenames', Reaper users can use wildcards to place media items in subfolders automatically. For instance: 

For a track named 'Guitar', specifying the recording path '`$Track\$Track-$recpass000`' would create a folder called 'Guitar' and place all takes recorded on that track into that folder with a name such as 'Guitar-001.wav' (I think $recpass should keep the names unique, but maybe a time stamp needs to be added, just to be sure).

Alternatively, you can also just use slashes in track names. So a user might prefer to name their guitar tracks:
Guitars\GuitarLeft
Guitars\GuitarRight
Guitars\Solo

Due to the slash, Reaper would create a 'Guitars' directory, and place GuitarLeft, GuitarRight and Solo subfolders into it, and then place the recorded track media items in the appropriate folder dependent on which track they were recorded.

Doing all this organisation means that there is a lot less in the root folder. Really only things like glued media items end up there.

### Request
All this to say - currently Playtime2 just places everything in the root of the project folder. I think it would be a nice feature to offer the user the option to choose a project subdirectory for all the clips to go into. For example, using the above nomenclature, being able to specify  '- Playtime2' as a subfolder would keep the project directory neat. Of course, allow the user to choose whatever directory name they wish, so it fits in with their own project folder organisation.

When Playtime2 clips are exported to the Arrange Window, Reaper does not do any automatic naming or moving. It only occurs when items are directly recorded onto the Arrange Window. Just to be clear, I think this is fine - the clip sources can be left where they are after exporting, and it isn't necessary for Playtime2 to work with the automatic media paths and wildcards set in Reaper; none of that stuff is part of this request.

I don't know how Linux or Mac handle their Reaper project directories, or what effect this request would have on those users.

**Comment by helgoboss** [comment]
Woah, thanks for the long explanation. Will think about this.
